learn/app-development/deployment/building-with-maven.md

```diff
@@ -3,64 +3,104 @@ title: "Building Project with Maven"
+|npm|6.4.1|
 |Ant|1.10.7|
 |Git| 2.26.2|
 
-## Generating a War file
 
-1. Export the project as zip.
+## Go to Source Location
 
-[![](/learn/assets/ExportProjectasZip.png)](/learn/assets/ExportProjectasZip.png)
+- WaveMaker Project sources can be fetched in two ways.
 
-2. Extract the downloaded zip file.
+### Clone VCS Repository
 
-3. Get the project location path.  
+- Clone Project into your pipeline or your local machine using git clone `<repo_url>`.
+- You can clone only if you have configured custom VCS or push to your own repo (in both WaveMaker Enterprise Setups, WaveMaker Online Teams). For more information, see [Configure Custom VCS](/learn/app-development/deployment/build-options).
 
-4. In the command line, go to the project path. See the image below.
+### Download Zip from Studio
+
+- Export the project as zip.
+
+[![export project](/learn/assets/ExportProjectasZip.png)](/learn/assets/ExportProjectasZip.png)
+
+- Extract the downloaded zip file.
+
+- Get the project location path.  
+
+- In the command line, go to the project path. See the image below.
 
 ```shell
 cd <location>
 ```
 
-[![](/learn/assets/LocateProjectIncmdline.png)](/learn/assets/LocateProjectIncmdline.png)
+[![locate project](/learn/assets/LocateProjectIncmdline.png)](/learn/assets/LocateProjectIncmdline.png)
 
-5. Run the following command where `<profileName>` should either be **development** or **deployment**.
+## Choose Profile to Build
 
+- WaveMaker project has two default profiles, including **development** and **deployment**. Prefix the profile name with a **P**. If you do not prefix the profile name; the system selects a **development** profile by default. You can add Custom Profiles from the **[Config Profiles](/learn/app-development/deployment/configuration-profiles)** section in the **Project Settings** options.
+- Use `<ProfileName>` in the next sections from  **default_profile/custom_profile**.
+
+## Build War file
+
+- Use the below command to do a maven build with the profile. 
+
+```shell
+mvn clean install -P<ProfileName>
 ```
-mvn clean install -PprofileName
+
+**For example**
+
+```shell
+mvn clean install -Pdeployment
 ```
 
-![mvn install](/learn/assets/enter-mvn-install-cmd-and-profilename.png)
+- This command will generate a `project war`  file in the target directory. The `project war`  file has both frontend artifacts (HTML, CSS, JS, images, etc), and backend artifacts (Java Classes).
+- This war file can be deployed into any webserver like Tomcat. For more information, see [App Deployment to Tomcat](/learn/how-tos/wavemaker-application-deployment-tomcat).
+
+## Build War file and Static Content to Deploy them Separately
+
+- WaveMaker app is consists of frontend artifacts (HTML, CSS, JS, images, etc), and backend artifacts (Java Classes). It is recommended to host frontend artifacts in Static Content Servlet like nginx, apache, etc, or Content Delivery Network (CDN), and backend artifacts can be hosted on any web server like Tomcat.
```

Comment:
Also 

static content server like nginx

not 

static content **servlet** like nginx